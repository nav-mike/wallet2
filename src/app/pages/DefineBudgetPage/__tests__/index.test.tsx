import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { DefineBudgetPage } from '..';

const shallowRenderer = createRenderer();

describe('<DefineBudgetPage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<DefineBudgetPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
