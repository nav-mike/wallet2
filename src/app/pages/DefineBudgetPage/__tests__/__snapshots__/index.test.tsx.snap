// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DefineBudgetPage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Define budget
    </title>
    <meta
      name="Define budget page"
    />
  </Helmet>
  <NavBar />
  <PageWrapper>
    <FlexContainer>
      <ColumnsContainer>
        <H1>
          Define budget
        </H1>
        <form>
          <FormGroup>
            <Label>
              Current budget value is 
              500
               EUR
            </Label>
          </FormGroup>
          <FormGroup>
            <Label>
              Budget value
            </Label>
            <Input
              min="0"
              type="number"
            />
          </FormGroup>
          <FormGroup>
            <SubmitButton
              type="button"
            >
              Submit
            </SubmitButton>
            <CancelButton
              href="/"
            >
              Cancel
            </CancelButton>
          </FormGroup>
        </form>
      </ColumnsContainer>
    </FlexContainer>
  </PageWrapper>
</React.Fragment>
`;
